src/packages/index.gleam

```diff
@@ -141,6 +141,22 @@ create table if not exists hidden_packages (
   name text primary key
 ) strict;
 
+create view if not exists retired_package_ids as
+  -- A package is retired if its latest release is retired
+  select package_id from (
+    select
+      package_id
+      , retirement_reason
+    from
+      releases
+    group by
+      package_id
+    having
+      max(inserted_in_hex_at)
+  )
+  where
+    retirement_reason is not null;
```

Comment:
Could you instead make a `non_retired_packages` view? I think that would be easier to use and it wouldn't have to read the entire table to filter a single package. Thank you

Reply:
It is done! (I learned some new things about sqlite along the way but it seems to work).

I had to adjust the text search query because sqlite views don't have `rowid`s, but the search seems to behave the same way as it does in production.